app/views/monthly_statistics/show.html.erb

```diff
@@ -3,79 +3,84 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      Applicants for initial teacher training courses: 2021 to 2022
+      Initial teacher training applications data for courses starting in the <%= @academic_year_name %> academic year
     </h1>
 
-    <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">Statistics up to midnight 30 November 2021</p>
-    <p class="govuk-body govuk-!-font-size-16">Next update on 3 January 2022</p>
-
     <div class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</div>
```

Comment:
``
govuk-!-font-size-20
``

Reply:
Actually this should remain `govuk-!-font-size-16` - the prototype is wrong. will fix. 🤦